website/.docusaurus/client-modules.js

```diff
@@ -1,7 +1,7 @@
 export default [
-  require("C:\\Users\\Angelo CG\\Desktop\\migracion guide.eoscostarica.io\\guide.eoscostarica.io\\website\\node_modules\\remark-admonitions\\styles\\infima.css"),
-  require("C:\\Users\\Angelo CG\\Desktop\\migracion guide.eoscostarica.io\\guide.eoscostarica.io\\website\\node_modules\\remark-admonitions\\styles\\infima.css"),
-  require("C:\\Users\\Angelo CG\\Desktop\\migracion guide.eoscostarica.io\\guide.eoscostarica.io\\website\\node_modules\\infima\\dist\\css\\default\\default.css"),
-  require("C:\\Users\\Angelo CG\\Desktop\\migracion guide.eoscostarica.io\\guide.eoscostarica.io\\website\\node_modules\\@docusaurus\\theme-classic\\src\\prism-include-languages"),
-  require("C:\\Users\\Angelo CG\\Desktop\\migracion guide.eoscostarica.io\\guide.eoscostarica.io\\website\\src\\css\\custom.css"),
+  require("C:\\Users\\Angelo CG\\Desktop\\guide.eoscostarica.io\\website\\node_modules\\remark-admonitions\\styles\\infima.css"),
```

Comment:
what are these absolute references to you local machine for?

Reply:
Eso no es de este pull request, ocurrió por el problema que tengo con mi repositorio de githup, se arrastro desde un pull request anterior era algo que ya esta solucionado no debería dar problema.